### PR TITLE
New README and guidelines

### PR DESCRIPTION
**Pull request by TheRealKeto**
This PR provides a much cleaner README, removing all references of documentation of the project, since that's been moved to the project's [wiki page](https://github.com/ProcursusTeam/Procursus), while also providing cleaner contribution guidelines.

I probably made a few typos here and there; just tell me through a PR review and I'll change it when I can.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,94 +1,32 @@
...
+<p align="center">
+    <h1 align="center">Procursus</h1>
+    <p align="center">
+        <a href="https://github.com/ProcursusTeam/Procursus/wiki">Documentation</a> •
+        <a href="https://procursus.creator-spring.com/">Merchandise</a>
+    </p>
+</p>
 
-A new, powerful cross-compilation system for *OS devices coupled with an APT repo.
+A new, powerful, cross-compilation *OS boostrap. At its core, Procursus is a build-system that provides a large set of consistently up-to-date *nix tools cross compiled to work on Darwin based platforms.
```
Comment (quiprr) [inline README.md:9]: misspelled bootstrap

**Review by quiprr — CHANGES_REQUESTED** [review]

**Comment by m1stadev** [comment]
readme dev strikes again

File: README.md (hunk 1 above)
Comment (TheRealKeto) [reply README.md]: Resolved with latest PR commit.

**Comment by CRKatri** [comment]
Could you transfer the wiki to be in a sub folder of the repo or even a separate repo so that the wiki can be accessed easily.

**Comment by TheRealKeto** [comment]
No. I see no point in that. Just visit the website; the transition was done so this wouldn't be a thing.

The wiki itself can be locally clone too, in case you really need the files.

**Comment by CRKatri** [comment]
It also allows us to make the wiki read only and have PRs for it.

**Comment by TheRealKeto** [comment]
The wiki is already read only. PRs to the wiki is something that would have low traffic.

**Comment by TheRealKeto** [comment]
@CRKatri Since you asked for it, [here you go,](https://github.com/TheRealKeto/procursus-docs)

**Comment by CRKatri** [comment]
It’s missing _Sidebar.md, it should be in the ProcursusTeam Org, and you should setup something similar to https://github.com/theos/theos-wiki/blob/main/.github/workflows/publish-wiki.yml. Personal I also think it’d be better as a docs sub folder in the main repo, but that’s up to @Diatrus.

**Comment by Diatrus** [comment]
I agree with that sentiment @CRKatri

**Comment by TheRealKeto** [comment]
The sidebar is irrelevant to the actual documentation; I pushed it under my own Github profile because I can't create repositories under the org, and doing a workflow is up to Hayden.

**Pull request merged**